apps/explorer/vite.config.ts

```diff
@@ -19,7 +19,11 @@ process.env.VITE_VERCEL_ANALYTICS_ID = process.env.VERCEL_ANALYTICS_ID;
 
 // https://vitejs.dev/config/
 export default defineConfig({
-    plugins: [react(), svgr()],
+    plugins: [react(), svgr()] as any,
+    test: {
+        globals: true,
```

Comment:
Why do we need to set this?

Reply:
I  will remove it

Reply:
will not work without it

Reply:
From Vitest doc `By default, vitest does not provide global APIs for explicitness. If you prefer to use the APIs globally like Jest, you can pass the --globals option to CLI or add globals: true in the config.`

Reply:
I know what it does, I'm asking why it fails without defining it.

Reply:
I see, when get `getByTestId` method from the render fun `const { getByTestId } = render`, it fails without global set to true. `TestingLibraryElementError: Unable to find an element by ...`  but works importing from `testing-library/dom`

Reply:
got it, so it's because react testing library requires it.